core/src/main/java/org/ironjacamar/core/workmanager/WorkManagerCoordinator.java

```diff
@@ -416,6 +416,19 @@ public synchronized WorkManager createWorkManager(String id, String name)
             if (dwm.getTransport() != null)
             {
                dwm.getTransport().register(new Address(wm.getId(), wm.getName(), dwm.getTransport().getId()));
+               try
+               {
+                  ((DistributedWorkManager) wm).getTransport().startup();
+               }
+               catch (Throwable t)
+               {
+                  log.debugf("Unable to start the DWM Transport ID:",
+                        ((DistributedWorkManager) wm).getTransport().getId());
+                  if (trace)
+                  {
+                     log.trace("Unable to start DWM Transport", t);
+                  }
+               }
```

Comment:
I think we need to make sure that a transport is associated in case of a DWM instance, and if any errors occur the ISE is thrown. The logging should be converted to ERROR